subsys/bluetooth/controller/ll_sw/ull_filter.c

```diff
@@ -470,7 +513,138 @@ uint8_t ll_priv_mode_set(bt_addr_le_t *id_addr, uint8_t mode)
 
 	return 0;
 }
+#endif /* CONFIG_BT_CTLR_PRIVACY */
+
+#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC)
+uint8_t ll_pl_size_get(void)
+{
+	return PL_SIZE;
+}
+
+uint8_t ll_pl_clear(void)
+{
+	/* FIXME: Check and fail if Periodic Advertising Create Sync is pending.
+	 */
+
+	pl_clear();
+
+	return 0;
+}
+
+uint8_t ll_pl_add(bt_addr_le_t *addr, uint8_t sid)
```

Comment:
For this and the following functions: 

```suggestion
uint8_t ll_pl_add(const bt_addr_le_t *addr, uint8_t sid)
```

Reply:
done